static/src/javascripts/projects/common/modules/article/progress.js

```diff
@@ -0,0 +1,36 @@
+define([
+    'common/utils/$',
+    'common/utils/mediator'
+], function (
+    $,
+    mediator
+) {
+    var articleHeight;
+
+    return {
+        init: function () {
+            // Check if progress bar is present
+            if ($('.progress').length) {
```

Comment:
I'm wary of using classes as springboards for scripted functionality. It's not obvious, looking at the template, how the component is instantiated, or that refactoring the styles will break the functionality. It also makes it awkward to re-use the behaviour without re-using the styles.

I would use a `js-progress-bar` class or the like, instead, in the same manner as existing components like `js-site-message`, `js-sport-tabs`, etc. etc.